Debian #361114; from upstream CVS; prevents the window from growing of one line
when there is one tab remaining.

===================================================================
RCS file: /cvs/gnome/gnome-terminal/src/terminal-window.c,v
retrieving revision 1.128
retrieving revision 1.129
diff -Nur gnome-terminal-2.14.2/src/terminal-window.c gnome-terminal-2.14.2.new/src/terminal-window.c
--- gnome-terminal-2.14.2/src/terminal-window.c	2006-03-04 07:21:01.000000000 +0100
+++ gnome-terminal-2.14.2.new/src/terminal-window.c	2007-03-18 10:54:49.000000000 +0100
@@ -1815,6 +1815,7 @@
                                TerminalScreen  *screen,
                                TerminalWindow  *window)
 {  
+  gboolean single;
   /* Called from terminal_notebook_move_tab() */
   if (find_screen (window, screen) != NULL) {
     g_assert (terminal_screen_get_window (screen) == window);
@@ -1846,6 +1847,11 @@
   reset_tab_menuitems (window);
   update_tab_sensitivity (window);
 
+  /* The tab bar may have disappeared */
+  single = g_list_length (window->priv->terms) == 1;
+  if (single)
+    terminal_window_set_size (window, window->priv->active_term, TRUE);
+
   /* Close window if no more terminals */
   if (window->priv->terms == NULL)
     gtk_widget_destroy (GTK_WIDGET (window));
